assignment2.py

```diff
@@ -0,0 +1,155 @@
+class ListNode:
+    def __init__(self, value):
+        self.next = None
+        self.value = value
+        
+def string_compression(s):
+    count = 1
+
+    compressed = []
```

Comment:
Is there a reason you didn't make this a string instead of a list? You can append to a string with `+`. The only reason to use a list here would be if you were using list comprehension to do this in one line.